src/Raven.Server/Web/System/StudioHandler.cs

```diff
@@ -152,6 +153,18 @@ public Task FavIcon()
         [RavenAction("/auth-error.html", "GET", AuthorizationStatus.UnauthenticatedClients)]
         public Task StudioAuthError()
         {
+            var feature = HttpContext.Features.Get<IHttpAuthenticationFeature>() as RavenServer.AuthenticateConnection;
+            var authStatus = feature?.Status;
+            
+            if (authStatus is RavenServer.AuthenticationStatus.ClusterAdmin or
```

Comment:
@Danielle9897 please fix syntax and use older version.

Reply:
done